pddlgym/tests/run_all_tests.py

```diff
@@ -0,0 +1,11 @@
+# import os
```

Comment:
remove this file or uncomment?

Reply:
Oops, will remove 😅

Reply:
> > Thanks so much for doing this. Looks broadly good but I want to take a closer look before merging and @ronuchit may too. To make it a little easier to review, would you mind splitting this into at least three PRs:
> > 
> > 1. exposing the submodules
> > 2. changing the tests to unittests
> > 3. adding type annotations
> 
> Sure! I'll link those here once they exist and ping you.

Unittests: #68 

Expose submodules: #69 

Type annotations: #70